cmd/opdoc/opdoc.go

```diff
@@ -166,53 +166,78 @@ func stackMarkdown(op *logic.OpSpec) string {
 	return out + "\n"
 }
 
-func opToMarkdown(out io.Writer, op *logic.OpSpec, groupDocWritten map[string]bool) (err error) {
+func opToMarkdown(out io.Writer, specs []logic.OpSpec, groupDocWritten map[string]bool) (err error) {
 	ws := ""
-	opextra := logic.OpImmediateNote(op.Name)
+	if len(specs) == 0 {
+		return nil
+	}
+	newest := specs[len(specs)-1]
+	oldest := specs[0]
```

Comment:
If you keep the name `op` for `newest` you can minimize the diffs a bit.

You could also take references to the OpSpec in the slice using pointers to minimize copies.  Not a big deal since this is just doc generation code.